gsl_pages/templates/gsl_pages/coming_features.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+    <iframe src="https://lowly-energy-62c.notion.site/ebd/18f765c7b6a480928d97d97dc20e2df8"
+            width="100%"
+            height="600"
+            frameborder="0"
+            allowfullscreen />
+</iframe>
```

Comment:
RGAA 2.1 et 2.2 : les cadres doivent avoir un titre, et il doit être pertinent.

```suggestion
    <iframe src="https://lowly-energy-62c.notion.site/ebd/18f765c7b6a480928d97d97dc20e2df8"
            width="100%"
            title="Présentation des fonctionnalités à venir"
            height="600"
            frameborder="0"
            allowfullscreen />
</iframe>
```

Reply:
Bien vu !